packages/local_auth/README.md

```diff
@@ -157,20 +157,13 @@ manifest (or your own Activity class if you are extending the base class).
 Update your MainActivity.java:
 
 ```java
-import android.os.Bundle;
-import io.flutter.app.FlutterFragmentActivity;
-import io.flutter.plugins.flutter_plugin_android_lifecycle.FlutterAndroidLifecyclePlugin;
-import io.flutter.plugins.localauth.LocalAuthPlugin;
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.FlutterEngine;
+import io.flutter.embedding.android.FlutterFragmentActivity;
+import io.flutter.plugins.GeneratedPluginRegistrant;
+import io.flutter.embedding.android.FlutterActivity;
 
 public class MainActivity extends FlutterFragmentActivity {
-    @Override
```

Comment:
Please take a look at the Kotlin example. This should look similar.

Also, please update the example app too:

https://github.com/flutter/plugins/blob/20f95a6a34c3b1f00bdcf56d7352bcc33819a216/packages/local_auth/example/android/app/src/main/java/io/flutter/plugins/localauthexample/EmbeddingV1Activity.java#L14